app/Console/Commands/masto_chain.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Http\Controllers\Backend\Social\MastodonController;
+use App\Models\Status;
+use Illuminate\Console\Command;
+
+class masto_chain extends Command
+{
+    protected $signature = 'trwl:masto {status_id}';
+
+    public function handle()
+    {
+        $statusId = $this->argument('status_id');
+        $status   = Status::where('id', $statusId)->firstOrFail();
+
+        dd(MastodonController::getEndOfChain($status->user, $status->mastodon_post_id));
+
+        return Command::SUCCESS;
+    }
+}
```

Comment:
This looks like a testing command which should be removed before merging, right?